lib/ecs_rails_console/cli.rb

```diff
@@ -40,11 +46,11 @@ def aws_credentials
     end
 
     def config
-      @config ||= YAML.load_file(config_file_from_project)[environment] || {}
+      @config ||= YAML.load_file(CONFIG_FILE)[environment] || {}
     end
 
-    def config_file_from_project
-      "#{Dir.pwd}/config/ecs_rails_console.yml"
+    def ssh_user
+      config[:ssh_user] || 'root'
```

Comment:
Isn't root defined in the defaults file? Can we pull that from there instead of hard-coding it here?

Reply:
What is the user did not define a user in the configuration? In case that this option is omitted, the gem should fallback to `root`.

Reply:
Yes, but my question was that we've already defined the ssh user default value here: https://github.com/net-engine/ecs-rails-console/pull/5/files#diff-0573d4d48ab73f9ea06d8b5943984b1b0eb8460d9a0d0e139dd47899c96be9e4R11 . I had assumed we could re-use that default value here, but is that assumption wrong?

Reply:
You are correct. This is the default user set in the configuration template. If the user keeps it, yeah, it's redundant. But what if the client just remove this option from the config file? Then the value will be absent, and to prevent the tool to run ssh with a `null` user, fallback to `root`.

Reply:
Ah, I see what you're saying. Sounds good